javascript/modules/client/test/integration/client.test.ts

```diff
@@ -420,6 +423,122 @@ describe("Client", () => {
       expect(typeof withdrawAction).toBe("object");
       expect(withdrawAction.data).toBeInstanceOf(Uint8Array);
     });
+    it("Should create a client and generate a grant action", () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const grantParams: IPermissionParams = {
+        who: "0x1234567890123456789012345678901234567890",
+        where: "0x1234567890123456789012345678901234567890",
+        permission: Permissions.UPGRADE_PERMISSION,
+      };
+      const grantParams2: IPermissionParams = {
+        who: "0x0987654321098765432109876543210987654321",
+        where: "0x0987654321098765432109876543210987654321",
+        permission: Permissions.EXECUTE_PERMISSION,
+      };
+
+      const grantAction = client.encoding.grantAction(
+        grantParams,
+      );
+      const grantAction2 = client.encoding.grantAction(
+        grantParams2,
+      );
+      const decoder = new TextDecoder();
+
+      expect(typeof grantAction).toBe("object");
+      expect(grantAction.to).toMatch(/^0x[A-Fa-f0-9]{40}$/i);
```

Comment:
If the next check is an exact match, there is no need to check a generic regex here